test/test_um2netcdf.py

```diff
@@ -651,6 +662,379 @@ def test_fix_units_do_nothing_no_um_units(ua_plev_cube):
         assert ua_plev_cube.units == orig  # nothing should happen as there's no cube.units
 
 
+@pytest.fixture
+def lat_river():
+    # iris DimCoord imitating UM V7.3s river grid. Must have length 180.
+    lat_river_points = np.arange(-90., 90, dtype="float32") + 0.5
+    lat_river_coord = iris.coords.DimCoord(
+        points=lat_river_points,
+        standard_name=um2nc.LATITUDE
+    )
+    return lat_river_coord
+
+
+@pytest.fixture
+def lon_river():
+    # iris DimCoord imitating UM V7.3s river grid. Must have length 90.
+    lon_river_points = np.arange(0., 360., dtype="float32") + 0.5
+    lon_river_coord = iris.coords.DimCoord(
+        points=lon_river_points,
+        standard_name=um2nc.LONGITUDE
+    )
+    return lon_river_coord
+
+
+@pytest.fixture
+def lat_v_nd():
+    # iris DimCoord imitating imitating the real
+    # lat_v grid from ESM1.5 (which uses the New Dynamics grid).
+    lat_v_points = np.arange(-90.+0.5*D_LAT_N96, 90,
+                             D_LAT_N96, dtype="float32")
+
+    lat_v_nd_coord = iris.coords.DimCoord(
+        points=lat_v_points,
+        standard_name=um2nc.LATITUDE,
+    )
+    return lat_v_nd_coord
+
+
+@pytest.fixture
+def lon_u_nd():
+    # iris DimCoord imitating the real
+    # lon_u grid from ESM1.5 (which uses the New Dynamics grid).
+    lon_u_points = np.arange(0.5*D_LON_N96, 360, D_LON_N96, dtype="float32")
+
+    lon_u_nd_coord = iris.coords.DimCoord(
+        points=lon_u_points,
+        standard_name=um2nc.LONGITUDE,
+    )
+    return lon_u_nd_coord
+
+
+@pytest.fixture
+def lat_v_eg():
+    # iris DimCoord imitating the real
+    # lat_v grid from CM2 (which uses the End Game grid).
+    lat_v_points = np.arange(-90., 91, D_LAT_N96, dtype="float32")
+
+    lat_v_eg_coord = iris.coords.DimCoord(
+        points=lat_v_points,
+        standard_name=um2nc.LATITUDE
+    )
+    return lat_v_eg_coord
+
+
+@pytest.fixture
+def lon_u_eg():
+    # iris DimCoord imitating the real
+    # lon_v grid from CM2 (which uses the End Game grid).
+    lon_u_points = np.arange(0, 360, D_LON_N96, dtype="float32")
+
+    lon_u_eg_coord = iris.coords.DimCoord(
+        points=lon_u_points,
+        standard_name=um2nc.LONGITUDE
+    )
+    return lon_u_eg_coord
+
+
+@pytest.fixture
+def lat_standard_nd():
```

Comment:
Flagging another fixture naming instance here...

Reply:
Just realised 
```python
    lat_points_middle = np.arange(-88.75, 89., D_LAT_N96)
    lat_points = np.concatenate(([-90],
                                 lat_points_middle,
                                 [90.]
                                 ), dtype="float32")
```
is exactly the same as `lat_points_middle = np.arange(-90, 91., D_LAT_N96)`...